scripts/process_linnarsson.sh

```diff
@@ -96,21 +97,21 @@ process_linnarson_images() {
             wget "$PKLAB_URL/Nuclei_polyT.int16.sf.hdf5" -O "$HDF5_IN"
 
         RELEASE=${S3_TARGET//vitessce-data\//}
-        ZARR_OUT='linnarsson.images.zarr'
-        ZARR_URL="https://vitessce-data.storage.googleapis.com/$RELEASE/linnarsson/$ZARR_OUT"
+        TILES_URL="https://vitessce-data.storage.googleapis.com/$RELEASE/linnarsson/$ZARR_OUT"
 
         CMD="$BASE/python/img_hdf5_reader.py
             --hdf5 $HDF5_IN
             --sample 1
             --channels polyT,nuclei
             --json_file $JSON_OUT
-            --zarr_url $ZARR_URL"
+            --zarr_file $ZARR_OUT
+            --tiles_url $TILES_URL"
         echo "Running: $CMD"
         $CMD
     fi
 
-    TILES_BASE="$ZARR_OUT/pyramid"
-    TILES_PATH="$OUTPUT/$TILES_BASE"
+    TILES_PATH="$ZARR_OUT/pyramid"
+    echo "$TILES_PATH/01"
```

Comment:
Log statements are fine, but add some explanation of what this string is.

Reply:
This is vestigial. Removed.